### ISSUE TITLE
new Find Texts field on inner pages

### ISSUE THREAD

**Issue by berger-n** [buda-base/public-digital-library#897]
now the Person and Topic pages have a working *Find Texts* field as described in the figma, with results defaulting to a no keyword search :
- https://library-dev.bdrc.io/show/bdr:T3

  ![image](https://github.com/user-attachments/assets/b468c6d3-8e21-455a-9d1d-2b1541685303)

- https://library-dev.bdrc.io/show/bdr:P258

  ![simplescreenrecorder-2024-07-19_17 39 12 mkv](https://github.com/user-attachments/assets/18f09942-e1e0-4daa-a22e-88a069ea37ce)

---

this raises the same issue than the last point in https://github.com/buda-base/public-digital-library/issues/876#issuecomment-2238716936: for now the results are all *created by* the person, not *about* the person; same for the Topic (it uses `workIsAbout`, not `workGenre`)

**Comment by berger-n** [comment]
now using `associated_res` so https://library-dev.bdrc.io/show/bdr:P258 contains P258 as `workIsAbout` as well, thanks!
there's nothing in the general case of an instance though: https://library-dev.bdrc.io/show/bdr:MW3PD1002
should I remove it there?

![image](https://github.com/user-attachments/assets/ebafe49f-a171-4e54-a37a-c1a6a524e41c)

**Comment by berger-n** [comment]
ok! done: https://library-dev.bdrc.io/show/bdr:MW3PD1002

![image](https://github.com/user-attachments/assets/8bd3db77-59dd-4a10-90d5-8aee61f56c00)

**Comment by berger-n** [comment]
not sure what to think about the auto-completion returning keywords that have no result in that case: 
https://library-dev.bdrc.io/show/bdr:T3

![simplescreenrecorder-2024-07-22_18 00 44 mkv](https://github.com/user-attachments/assets/7fb9e0d1-171d-47a4-8cc7-ec0fd19c048e)

**Comment by eroux** [comment]
good point yes! maybe in that context we want to disable auto-complete... or maybe @roopeux you had an idea about scoped autocomplete?

**Comment by eroux** [comment]
let's move the discussion about the scoped autosuggest here: https://github.com/buda-base/autocomplete-prototype/issues/8
